content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -323,7 +323,7 @@ either be a snapshot file from a previous backup operation, or from a remaining
 Here is an example:
 
 ```shell
-ETCDCTL_API=3 etcdctl --endpoints 10.2.0.9:2379 snapshot restore snapshotdb
+ETCDCTL_API=3 etcdctl --endpoints 10.2.0.9:2379 --data-dir <data-dir-location> snapshot restore snapshotdb
```

Comment:
I think `etcdctl snapshot restore` is deprecated in favor of `etcdutl`. Can we use `etcdutl`? Otherwise, we'd have to update it again in a few months once we upgrade k8s to etcd 3.6

Reply:
https://github.com/etcd-io/etcd/tree/main/etcdctl#snapshot-restore-options-filename

Reply:
I think its good to update with etcdutl once it is removed.

Reply:
lets update it once it is removed.

Reply:
Let's update it know, before it's removed as none will remember to update it then. 

I also would propose to separate command call that uses endpoint vs data dir 
 like in https://github.com/kubernetes/website/pull/27372

Reason is that both flags specify destination, either send restore request to etcd or lay out directory with snapshot data to run etcd server on it.

Reply:
sure will update.